templates/ntp.conf.trusty.j2

```diff
@@ -0,0 +1,36 @@
+# /etc/ntp.conf, configuration for ntpd; see ntp.conf(5) for help
+
+driftfile /var/lib/ntp/ntp.drift
+
+statistics loopstats peerstats clockstats
```

Comment:
I'm still seeing these differences between this template and the configuration file used by Trusty by default:

```diff
diff --git a/templates/ntp.conf.trusty.j2 b/templates/ntp.conf.trusty.j2
index 9f7e8e5..7cb16db 100644
--- a/templates/ntp.conf.trusty.j2
+++ b/templates/ntp.conf.trusty.j2
@@ -2,6 +2,10 @@
 
 driftfile /var/lib/ntp/ntp.drift
 
+
+# Enable this if you want statistics to be logged.
+#statsdir /var/log/ntpstats/
+
 statistics loopstats peerstats clockstats
 filegen loopstats file loopstats type day enable
 filegen peerstats file peerstats type day enable
@@ -34,3 +38,17 @@ restrict -6 default kod notrap nomodify nopeer noquery
 # Local users may interrogate the ntp server more closely.
 restrict 127.0.0.1
 restrict ::1
+
+# Clients from this (example!) subnet have unlimited access, but only if
+# cryptographically authenticated.
+#restrict 192.168.123.0 mask 255.255.255.0 notrust
+
+
+# If you want to provide time to your local subnet, change the next line.
+# (Again, the address is an example only.)
+#broadcast 192.168.123.255
+
+# If you want to listen to time broadcasts on your local subnet, de-comment the
+# next lines.  Please do this only if you trust everybody on the network!
+#disable auth
+#broadcastclient
```

I know that all of it is commented out, but I like the idea of making the minimal amount of changes necessary.